src/main/java/mate/controller/RemoveDriverFromCarController.java

```diff
@@ -0,0 +1,29 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+public class RemoveDriverFromCarController extends HttpServlet {
```

Comment:
you didn't need to implement a such controller, remove it

Reply:
If we have an ability to add driver to car, we must have an ability to remove driver from car, do we?

Reply:
in a real project maybe, but in this one, you shouldn't have done that, you don't have a such task in the description